static/css/custom.css

```diff
@@ -130,4 +130,9 @@ img {
     height: 90px;
     width: 90px;
     padding: 5px;
-}
\ No newline at end of file
+}
+
+body.dark {
+  background-color: rgb(30,34,39);
+  color: #fff;
```

Comment:
```suggestion
  color: #000;
```

The `Read More` text appears white when dark mode is turned on.

Also changing the color to `#000` will have a side-effect of the `310 stories fetched ✨` text appearing black

Reply:
@Bhupesh-V 
pushing a css fix for `Read More`
also I can see the stories fetched turning white correctly

![image](https://user-images.githubusercontent.com/15111263/83947080-68e6aa80-a832-11ea-9807-23f35c7636ba.png)